src/main/java/uk/gov/di/ipv/cri/common/library/service/ConfigurationService.java

```diff
@@ -37,33 +41,52 @@ public enum SSMParameterName {
 
     @ExcludeFromGeneratedCoverageReport
     public ConfigurationService() {
-        this.clock = Clock.systemUTC();
-        this.ssmProvider = ParamManager.getSsmProvider();
-        this.secretsProvider = ParamManager.getSecretsProvider();
-        this.parameterPrefix =
-                Objects.requireNonNull(
-                        System.getenv("AWS_STACK_NAME"), "env var AWS_STACK_NAME required");
+        this(
+                ParamManager.getSsmProvider(
+                        SsmClient.builder().httpClient(UrlConnectionHttpClient.create()).build()),
+                ParamManager.getSecretsProvider(
+                        SecretsManagerClient.builder()
+                                .httpClient(UrlConnectionHttpClient.create())
+                                .build()),
+                getEnvironmentVariableValue("AWS_STACK_NAME"),
```

Comment:
change `getEnvironmentVariableValue` to `System.getenv` ( this env variable will always be present )